### DISCUSSION TITLE
希望能够增加openai的接口数量

### DISCUSSION THREAD

**Discussion by darkelfzdy** [lobehub/lobehub#3383] [Ideas]
希望能够增加openai的接口数量，目前语言模型中只有一个openai接口，希望能够自由添加多个openai兼容接口，这样就能自己添加多个ai代理了。可以参考沉浸式翻译插件，他们家就是可以自由添加不限数量的openai兼容接口。

**Comment by xingpingcn** [comment]
我记得lobe有计划支持这个特性，如果你着急的话试试oneapi

**Comment by darkelfzdy** [comment]
我正在使用one-api，但是我的需求更复杂，需要直连openai，也需要代理一些，所以需要多个openai接口，单靠one-api无法实现

**Reply by BrandonStudio** [reply]
我觉得你可以使用分层的多个one-api实现
对不同来源的模型可以指定不同的模型后缀

**Comment by arvinxx** [comment]
new ai provider 已经支持
